samples/GraphQL.DataLoader.Example/Program.cs

```diff
@@ -0,0 +1,43 @@
+using DataLoaderGql;
+using DataLoaderGql.GraphQl;
+using GraphQL;
+using Microsoft.EntityFrameworkCore;
+
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddScoped<SalespeopleByNameDataLoader>().AddScoped<CarsBySalespersonDataLoader>();
+// Add services to the container.
+// Learn more about configuring OpenAPI at https://aka.ms/aspnet/openapi
+builder.Services.AddDbContext<DealershipDbContext>();
+
+builder.Services.AddGraphQL(builder =>
+{
+    builder
+        .AddSystemTextJson()
+        .AddGraphTypes(typeof(CarsGraphType).Assembly)
+        .AddSchema<DealershipSchema>();
+});
+var app = builder.Build();
+
+app.UseGraphQL();
+app.UseGraphQLAltair();
+app.MapGraphQLAltair().AllowAnonymous();
+await app
+    .Services
+    .CreateAsyncScope()
+    .ServiceProvider
+    .GetRequiredService<DealershipDbContext>()
+    .Database
+    .MigrateAsync()
+    .ConfigureAwait(false);
```

Comment:
This code is missing a `using` around the scope that is created.  I suggest a slight modification to make it easier to understand:

```suggestion
await using (var scope = app.Services.CreateAsyncScope())
{
    var dbContext = scope.ServiceProvider.GetRequiredService<DealershipDbContext>();
    await dbContext.Database.MigrateAsync().ConfigureAwait(false);
}
```